src/test/java/seedu/address/model/assignment/AssignmentTest.java

```diff
@@ -107,4 +103,44 @@ public void equals() {
         assertFalse(ASSIGNMENT_CS3230_LAB.equals(editedCs3230Lab));
 
     }
+
+    @Test
+    void compareTo() {
```

Comment:
I trust that your variables are correct. looks like a wide variety of cases you have considered